modules/machine-node-custom-partition.adoc

```diff
@@ -0,0 +1,293 @@
+// Module included in the following assemblies:
+//
+// * machine_management/
+// * machine_management/
+// * post_installation_configuration/node-tasks.adoc
+
```

Comment:
Consider adding the metadata that Kathryn mentioned in the last team meeting up here.